project/Dependencies.scala

```diff
@@ -6,8 +6,16 @@ import buildinfo.BuildInfo
 
 object Dependencies {
   object Versions {
-    val scala212 = "2.12.10"
-    val scala213 = "2.13.1"
+    val scala212 = "2.12.11"
+    val scala213 = "2.13.2"
+
+    // Libs and plugins must share a version. The root project must use that (and set the crossScalaVersions
+    // as empty list)so each sub-project can then decide which scalaVersion and crossCalaVersions they use.
+    val commonScalaVersion = scala212
```

Comment:
Is this still needed/used?

Reply:
In the PR as it is now it's dead code.

But I just remembered why I added it in the first place: I wanted to add the comment. The rules and restrictions `sbt` uses to decide the default `scalaVersion` were unclear to me when working on this. 

So, for example, the PR now uses `scala212` in a couple of places on `build.sbt` which could be considered as `magic number`'s. If, instead, we used `commonScalaVersion` in `root`'s `scalaVersion := ...` on `build.sbt` it'd be more clear why `root` needs to use scala212.

I'll move the comment to `build.sbt` and remove the `val`.